hyppo/kgof/kernel.py

```diff
@@ -159,13 +158,14 @@ def eval(self, X, Y):
         ------
         K : a n1 x n2 Gram matrix.
         """
-        # (n1, d1) = X.shape
-        # (n2, d2) = Y.shape
-        # assert d1==d2, 'Dimensions of the two inputs must be the same'
+        (n1, d1) = X.shape
+        (n2, d2) = Y.shape
+        assert d1 == d2, "Dimensions of the two inputs must be the same"
         sumx2 = np.reshape(np.sum(X**2, 1), (-1, 1))
         sumy2 = np.reshape(np.sum(Y**2, 1), (1, -1))
         D2 = sumx2 - 2 * np.dot(X, Y.T) + sumy2
         K = np.exp(old_div(-D2, (2.0 * self.sigma2)))
+        # K = compute_kern(x=X._value, y=Y)
```

Comment:
Don't comment out unless this is coming back later. If so, indicate with a comment

Reply:
Got it.